### DISCUSSION TITLE
What is coherence?

### DISCUSSION THREAD

**Discussion by matthewmccarthy11** [life-itself/community#955] [Research]
As part of our cohere+ project, we would like to get an understanding of how the LI community and our project partners consider 'coherence'. Feel free to use examples, added context, or whatever else in this discussion forum.

**Comment by martinprogrammer** [comment]
@matthewmccarthy11  

Coherence (IMO) can be **aligning logic** or **incorporating different logics or views into a unified theory**. 
It's a process (or a state) of **coming together** (or harmonising) in order to **work more effectively, while acknowledging the differences and negotiating common ground.**   

Maybe it's also a process of making a **case for working together and agreeing the terms?**

We had an example today - the discussion was about the "schizoid structure of our relation to reality".

1) One party associated the disassociation from reality with profound changes in the reality itself.

2) The other interlocutor suggested that maybe we were under illusion that reality was what it wasn't
and our realisation of its true nature is what is causing the disassociation.

Coherence would be to re-examine the individual views and come back with a combined vision.

Does the above address the question at all?
